web/api/app/Controller/EventsController.php

```diff
@@ -83,10 +83,14 @@ public function view($id = null) {
 		}
 		$options = array('conditions' => array('Event.' . $this->Event->primaryKey => $id));
 		$event = $this->Event->find('first', $options);
+		$this->Event->id = $id;
+                $event_neighbors = $this->Event->find('neighbors');
 		
 		$path = $configs['ZM_DIR_EVENTS'].'/'.$this->Image->getEventPath($event).'/';
 
 		$event['Event']['BasePath'] = $path;
+		$event['Event']['Next'] = $event_neighbors['next']['Event']['Id'];
+                $event['Event']['Prev'] = $event_neighbors['prev']['Event']['Id'];	
```

Comment:
would this throw exception if next/prev are not available (for the last/first event?)